lib/awesome_bot/statuses.rb

```diff
@@ -18,6 +18,10 @@ def net_status(url, head)
       head ? net_head_status(url) : net_get_status(url)
     end
 
+    def status_is_redirected(status)
```

Comment:
FYI: Ruby convention is to call this method `status_is_redirected?`. Having the `?` conveys that it returns a boolean. But that's just convention. I'd go with `is_a_redirect_status?`, but that's just preference.